plugins/inputs/kube_inventory/kube_state.go

```diff
@@ -176,6 +190,7 @@ func init() {
 		return &KubernetesInventory{
 			ResponseTimeout: internal.Duration{Duration: time.Second * 5},
 			Namespace:       "default",
+			BearerToken:     "/run/secrets/kubernetes.io/serviceaccount/token",
```

Comment:
Won't this mean the BearerToken will always be set later during the checks?

I think what we will want to do is get rid of the priority system for the two bearer token option.  If they are both set then on initialization it should be an error.  Technically a breaking change, but I think minor enough impact to consider.

Reply:
Sorry, I thought I removed this line